pkg/karmadactl/join/join.go

```diff
@@ -127,9 +122,8 @@ func (j *CommandJoinOption) Validate(args []string) error {
 // AddFlags adds flags to the specified FlagSet.
 func (j *CommandJoinOption) AddFlags(flags *pflag.FlagSet) {
 	flags.StringVar(&j.ClusterNamespace, "cluster-namespace", options.DefaultKarmadaClusterNamespace, "Namespace in the control plane where member cluster secrets are stored.")
-
 	flags.StringVar(&j.ClusterContext, "cluster-context", "",
-		"Context name of cluster in kubeconfig. Only works when there are multiple contexts in the kubeconfig.")
+		"Context name of cluster in kubeconfig. Please make sure this context exists in the cluster, default use current context.")
```

Comment:
```suggestion
		"Name of cluster context in kubeconfig. The current context is used by default.")
```

Reply:
OK, but my e2e cannot pass. I'll check it later.

Reply:
It seems we also need to update the e2e, because it using the old process.